content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md

```diff
@@ -9,6 +9,7 @@ tags:
 - enterprise
 - disks
 - dashboard
+- data reporting
```

Comment:
The Enclosure screen shows the storage layout of disks/drives in the system, no data reporting at all, just the layout of the system if it is TrueNAS hardware.
```suggestion
```